installation/scripts/testing-common.sh

```diff
@@ -7,24 +7,42 @@ function context_arg() {
     fi
 }
 
+# retries are useful when api call can fail due to the infrastructure issue
+function executeKubectlWithRetries() {
+    local command="$1"
+    local retry=0
+
+    while [[ ${retry} -lt 5 ]]; do
+        result=$(${command})
+        if [[ $? -eq 0 ]]; then
+            return 0
+        else
+            echo ${result}
```

Comment:
try to isolate that in dedicated shell, and return from this function only one output and get rid of the global `result` variable